Marlin/src/inc/SanityCheck.h

```diff
@@ -376,6 +376,12 @@
   #error "USB_SD_DISABLED is now NO_SD_HOST_DRIVE. Please update your Configuration_adv.h."
 #elif defined(USB_SD_ONBOARD)
   #error "USB_SD_ONBOARD is obsolete. Disable NO_SD_HOST_DRIVE instead."
+#elif POWER_SUPPLY == 1
+  #error "Replace POWER_SUPPLY 1 by enabling PSU_CONTROL and setting PSU_ACTIVE_HIGH to 'false'."
+#elif POWER_SUPPLY == 2
+  #error "Replace POWER_SUPPLY 2 by enabling PSU_CONTROL and setting PSU_ACTIVE_HIGH to 'true'."
+#elif defined(POWER_SUPPLY)
```

Comment:
Wow. Great. That gives us the chance to completely get rid of the comment in the configs (// Set 'false' for ATX (1), 'true' for X-Box (2)).

Reply:
Indeed. A more general comment will be nicer.
As usual, I will follow up on the things you suggest that I should do as time allows.